src/components/NcIconSvgWrapper/NcIconSvgWrapper.vue

```diff
@@ -285,6 +285,11 @@ export default {
 		vertical-align: text-bottom;
 	}
 
+	// Icon svg wrapper
+	span {
+		line-height: 0;
+	}
```

Comment:
I'd say that the root issue is that this span is a text (`inline`) element, which applies the `line-height`.

Instead of overriding text height to 0, I'd propose to make it a block or inline-block element.

I am unsure `line-height` won't be overridden by some a11y extension or have other browser-specific issues.

So I'd consider it a more risky solution than solving the root issue.

Reply:
Block does not help as it still considers the line height.
Either set line height like here or use either `grid` (makes no sense) or `flex` which will  fit content.

Reply:
> Block does not help as it still considers the line height.

Hmm, indeed

> `flex` which will fit content.

Yep, `flex`/`inline-flex`/`grid` works fine.

Or `display: contents`

> makes no sense

IMO, it makes more sense, as it sets a more relevant box type, than changing line-height of non-text 👀